packages/oas-to-har/src/index.js

```diff
@@ -6,8 +6,15 @@ const parseDataUrl = require('parse-data-url');
 
 const configureSecurity = require('./lib/configure-security');
 const removeUndefinedObjects = require('./lib/remove-undefined-objects');
+const formatStyle = require('./lib/style-formatting');
 
 function formatter(values, param, type, onlyIfExists) {
+  if (param.style) {
+    // Note: Technically we could send everything through the format style and choose the proper default for each
```

Comment:
This is a comment for potential future cleanup. Technically non-styled formats are coming through with a default style, so we could push all parameters through this style logic instead of branching.

Reply:
Just a note for future reference, the spec actually very clearly defines what all those defaults are.